test/unit/webdriver/log_test.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+import httpretty
+
+from appium.webdriver.webdriver import WebDriver
+from test.unit.helper.test_helper import appium_command, get_httpretty_request_body, ios_w3c_driver
+
+
+class TestWebDriverLog(object):
+    @httpretty.activate
+    def test_get_log_types(self):
```

Comment:
would it be possible to use pytest style and write test methods at the module level instead of putting them into a class?